.github/workflows/examples.yml

```diff
@@ -53,7 +53,10 @@ jobs:
         if [ ${{ matrix.python-version }} = 3.6 ]; then
           IGNORES='chainermn_.*|fastai*|rapids_.*|botorch_.*'
         elif [ ${{ matrix.python-version }} = 3.8 ]; then
+          IGNORES='chainermn_.*|dask_ml_.*|keras_.*|tensorboard_.*|tensorflow_.*|tfkeras_.*|fastai*|allennlp|rapids_.*'
```

Comment:
I think this line is duplicated with the next line. In addition, `allennlp` is supported by Python 3.8 according to #2226.  Could you remove this line?
```suggestion
```